src/pagination/pagination.component.ts

```diff
@@ -129,12 +135,19 @@ export interface PaginationTranslations {
 						[(ngModel)]="currentPage">
 						<option *ngFor="let page of pageOptions; let i = index;" class="bx--select-option" [value]="i + 1">{{i + 1}}</option>
 					</select>
-					<ibm-icon-chevron-down16
+					<svg
```

Comment:
Can we use the directive form of the icon here? `<svg ibmIconChevronDown16></svg>`?

Reply:
The directive form worked! Made the changes.